### ISSUE TITLE
add debian and ubuntu images

### ISSUE THREAD

**Issue by juhp** [toolbox-images/toolbox-images#1]
It would be nice to have ubuntu images available here:

@Jmennius and @kronenpj would you be interested in contributing to this?

Also Debian images: @andrewaylett and @bureado, interested?

**Comment by Jmennius** [comment]
I expect that upstream will accept them (eventually) since they have indicated that they are interested... I guess I'll ping them.

**Comment by juhp** [comment]
Yes, that is also my hope - the idea of this repo is to have a place to easily collect and collaborate on toolbox OS image info rather than having it all scattered across various open and closed PRs and forks, etc. Also there are more obscure OS's and configurations that might never get supported by the main toolbox...

It is not meant at as substitute for upstream support.
